Tests/XCUITests/HomePageSettingsUITest.swift

```diff
@@ -221,6 +221,15 @@ class HomePageSettingsUITests: BaseTestCase {
 //        waitForExistence(app.cells.staticTexts["Example Domain"], timeout: 3)
     }
 
+    func testRecentlySaved() {
+        // Preconditons: Create 10 bookmarks & add 10 items to reading list
```

Comment:
The test is failing on Bitrise, I think you may need something like:
`waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT)`
before this line